### PR TITLE
feat [added product listing page]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Added product listing page and also Product Data Provider Context  to make products available all over the application.
Product -listing react

### CODE REVIEW

File: src/Components/Product-listing/ProductListing.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from "react";
+import { useProductDataProvider } from "../../Context/ProductDataContext/ProductDataProvider";
+import { ProductCard } from "./ProductCard";
+import { Filter } from "./Filter/Filter";
+import "./styles.css";
+export const ProductListing = () => {
+  const { productList } = useProductDataProvider();
+  useEffect(() => {
+    console.log(productList);
+  }, []);
```
Comment (prachi-sahani) [inline src/Components/Product-listing/ProductListing.js]: looks like unnecessary code

**Review by prachi-sahani — COMMENTED** [review]
the code looks clean

Suggestion 
- remove console.log
- add to cart and like button can be put into one folder

**Review by RakshithBhat03 — APPROVED** [review]
The code is clean and well structured. Please implement the suggestions provided by Prachi.

**Review by Bharatrathod-81 — APPROVED** [review]
looks good

**Pull request merged**